server/tests/utils/article.test.js

```diff
@@ -0,0 +1,29 @@
+import { expect } from 'chai';
+import { describe } from 'mocha';
```

Comment:
I don't think you have to `import { describe } from mocha`

Reply:
Alright thanks, I'll make an edit to the file.